packages/coordinator/src/components/Table/Table.stories.tsx

```diff
@@ -0,0 +1,263 @@
+import Table, { TableProps } from "./Table";
+import { BloodType, Donor } from "@zm-blood-components/common";
+import { searchDonorColumns } from "../../screens/searchDonorsScreen/SearchDonorsScreen";
+
+// eslint-disable-next-line import/no-anonymous-default-export
+export default {
+  component: Table,
+  title: "Components Row Table",
+};
+
+const rows: CardTableRow<Donor>[] = [
+  {
+    rowData: {
+      lastName: "Zcv",
+      birthDate: "",
+      id: "DOH51a9KJ2VJBpZsSaFe72dHygr2",
+      bloodType: BloodType.O_PLUS,
+      firstName: "Cxv",
+      phone: "0534251426",
+      email: "sfg@ghxf.com",
+    },
+  },
+  {
+    rowData: {
+      email: "tt@test.com",
+      firstName: "Lebron",
+      id: "exjX1IXpB5e1oBIV1Dcmgj9f2nw1",
+      bloodType: BloodType.NOT_SURE,
+      birthDate: "",
+      lastName: "James",
+      phone: "0524498853",
+    },
+  },
+  {
+    rowData: {
+      email: "nirztest22@gmail.com",
+      lastName: "8",
+      bloodType: BloodType.B_MINUS,
+      birthDate: "",
+      phone: "0544336673",
+      firstName: "Nir",
+      id: "QoJEdyCYPJSDM4UyMn2W8qOw15j2",
+    },
+  },
+  {
+    rowData: {
+      lastName: "sssd",
+      email: "aa@a.com",
+      birthDate: "",
+      phone: "0543334442",
+      id: "C1OHjLa4h5MMImCLiMei2stQkaA3",
+      bloodType: BloodType.AB_MINUS,
+      firstName: "aaa",
+    },
+  },
+  {
+    rowData: {
+      phone: "0528444444",
+      lastName: "hos1",
+      firstName: "cor1",
+      bloodType: BloodType.A_MINUS,
+      id: "XkqyI9ETwicD4TzX2qnFF9rBXHP2",
+      email: "nirztest8@gmail.com",
+      birthDate: "",
+    },
+  },
+  {
+    rowData: {
+      lastName: "zm_test",
+      email: "cor_zm_test@a.com",
+      id: "AIKuXNaZqITgyFppQdZGCqrgW7d2",
+      birthDate: "",
+      firstName: "cor_test",
+      phone: "0542266558",
+      bloodType: BloodType.A_PLUS,
+    },
+  },
+  {
+    rowData: {
+      birthDate: "",
+      bloodType: BloodType.O_PLUS,
+      phone: "0528499999",
+      lastName: "hos1",
+      id: "yMwq3NQBtlaoy2LAerdgBESubM13",
+      email: "nirztest14@gmail.com",
+      firstName: "donor1",
+    },
+  },
+  {
+    rowData: {
+      id: "8doOcKEAvsbNV5RMnzU0Tew9ETo2",
+      bloodType: BloodType.A_PLUS,
+      firstName: "qwer",
+      phone: "0534323432",
+      lastName: "ty",
+      email: "qwer@gmail.com",
+      birthDate: "",
+    },
+  },
+  {
+    rowData: {
+      email: "rakad@gmail.com",
+      birthDate: "",
+      id: "2AL2kLgUSFhhydNdm0L64N9TEC12",
+      lastName: "ka",
+      phone: "0501234567",
+      firstName: "ra",
+      bloodType: BloodType.O_PLUS,
+    },
+  },
+  {
+    rowData: {
+      email: "tr@test.com",
+      birthDate: "",
+      firstName: "tr tester",
+      id: "nS8qDFTA2PaIvZKdVzo51UATtt82",
+      bloodType: BloodType.A_MINUS,
+      lastName: "333",
+      phone: "21112234",
+    },
+  },
+  {
+    rowData: {
+      bloodType: BloodType.A_PLUS,
+      phone: "0525938307",
+      email: "harelglik@gmail.com",
```

Comment:
Please obfuscate all the data here.
i.e. change names, emails and phone numbers to not have any real info (like this one above)

Reply:
Do this for all the hard coded data